packages/common-scripts/index.d.ts

```diff
@@ -86,4 +168,114 @@ export declare const secp256k1Blake160Multisig: {
    * @returns lock script args
    */
   multisigArgs(serializedMultisigScript: string): string,
+
+    /**
+   * Inject capacity from `fromInfo` to target output.
+   *
+   * @param txSkeleton
+   * @param outputIndex
+   * @param fromInfo
+   * @param options
+   */
+  injectCapacity(
+    txSkeleton: TransactionSkeleton,
+    outputIndex: number,
+    fromInfo: FromInfo,
+    options: {
+      config: Config,
+    },
+  ): TransactionSkeleton,
+
+  /**
+   * Setup input cell infos, such as cell deps and witnesses.
+   *
+   * @param txSkeleton
+   * @param inputIndex
+   * @param fromInfo
+   * @param options
+   */
+  setupInputCell(
+    txSkeleton: TransactionSkeleton,
+    inputIndex: number,
+    fromInfo: FromInfo | undefined,
+    options: {
+      config: Config,
+    },
+  ): TransactionSkeleton,
+}
+
+export declare const dao: {
+  /**
+   * deposit a cell to DAO
+   *
+   * @param txSkeleton
+   * @param fromInfo
+   * @param toAddress deposit cell lock address
+   * @param amount capacity in shannon
+   * @param options
+   */
+  deposit(
+    txSkeleton: TransactionSkeleton,
+    fromInfo: FromInfo,
+    toAddress: Address,
+    amount: bigint,
+    options: {
+      config: Config,
+    },
+  ): Promise<TransactionSkeleton>,
+
+  /**
+   * list DAO cells,
+   *
+   * @param cellProvider
+   * @param fromAddress
+   * @param cellType "deposit" or "withdraw", can using "" to list all.
```

Comment:
Whenever you need comment like this, please create a new type with the 2 possible values.

Reply:
OK, updated now.